src/main/scala/uk/gov/nationalarchives/aws/utils/StepFunctionUtils.scala

```diff
@@ -16,8 +17,18 @@ class StepFunctionUtils(client: SfnAsyncClient) {
 
     IO(client.sendTaskSuccess(request)).futureLift
   }
+
+  def sendTaskFailureRequest(taskToken: String, error: Option[String]): IO[SendTaskFailureResponse] = {
```

Comment:
Does this need to be optional? It seems like it would be good to force the client to supply an error message that has more context.

Reply:
It doesn't. I was just concerned about the client not being able to supply an error message, depending on what the problem was. But you're correct, it would be better to force the client to supply something.